src/components/Chat/UserInput/styles.scss

```diff
@@ -30,9 +30,37 @@
   min-width: 70%;
```

Comment:
User input line needs more padding and icons are a little packed with mobile. Maybe even smaller icons or smaller text field?

Reply:
Good point. I made the icons a bit smaller and adjusted the size of text field. I also changed the position of the input-field to fixed. The iPhone 5 scrolling problem was due to the header which is too big for the page. I fixed this by changing font size for header to use vw. The browser support for older browsers may not be great, is that a problem? https://caniuse.com/#feat=viewport-units